ydb/core/viewer/viewer.cpp

```diff
@@ -198,6 +198,92 @@ class TViewer : public TActorBootstrapped<TViewer>, public IViewer {
     TString GetHTTPFORBIDDEN(const NMon::TEvHttpInfo* request) override;
     TString GetHTTPNOTFOUND(const NMon::TEvHttpInfo* request) override;
 
+    bool CheckAccessAdministration(const NMon::TEvHttpInfo* request) override {
+        if (!KikimrRunConfig.AppConfig.GetDomainsConfig().GetSecurityConfig().GetEnforceUserTokenRequirement()) {
```

Comment:
the conditions are duplicated here. It looks strange to me `!KikimrRunConfig.AppConfig.GetDomainsConfig().GetSecurityConfig().GetEnforceUserTokenCheckRequirement()`
if the token should not be verified, is it so important to look at its presence? `request->UserToken.empty()`

Reply:
GetEnforceUserTokenRequirement() and GetEnforceUserTokenCheckRequirement() are two different options